ZeroWasteFriendly/Scenes/EcoMap/Displayable/ZeroWastePlaceAnnotation.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ZeroWastePlaceAnnotation.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 21/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import Foundation
+import MapKit
+
+final class ZeroWastePlaceAnnotation: NSObject, MKAnnotation {
+
+    let title: String?
+    let subtitle: String?
+    let category: ZeroWasteCategory
+    let coordinate: CLLocationCoordinate2D
+
+    init(place: ZeroWastePlace) {
+        self.title = place.name
+        self.subtitle = place.description
+        self.category = place.category
+        self.coordinate = CLLocationCoordinate2D(latitude: place.latitude,
+                                                 longitude: place.longitude)
+        super.init()
+    }
+
+    var markerTintColor: UIColor? {
```

Comment:
color and image are related wth ZeroWasteCategory, so why is it in PlaceAnnotation?

I'd go: var markerTintColor: UIColor? { return category.markerTintColor }